docs/integrations/platforms/kubernetes-injector.mdx

```diff
@@ -63,6 +63,7 @@ The Infisical Agent Injector supports the following annotations:
 
   - `init`: The init method will create an init container for the pod that will render the secrets into a shared volume mount within the pod. The agent init container will run before any other containers in the pod runs, including other init containers.
   - `sidecar`: The sidecar method will create a sidecar container for the pod that will render the secrets into a shared volume mount within the pod. The agent sidecar container will run alongside the main container in the pod. This means that the secrets rendered will always be in sync with your Infisical secrets.
+  - `sidecar-init`: The sidecar-init method will create the init container and the sidecar container from the other two methods.
```

Comment:
currently i don't understand why i would want to use this. adding more contex here would help. is the idea is that it will run a init container then keep it alive?